src/earn/poolInfoScreen/BeforeDepositBottomSheet.tsx

```diff
@@ -226,48 +297,76 @@ export default function BeforeDepositBottomSheet({
     depositTokenId: pool.dataProps.depositTokenId,
   }
 
+  const showCrossChainSwap = canSwapDeposit && hasTokensOnOtherNetworks
+  const showSwap = !canSwapDeposit && (hasTokensOnSameNetwork || hasTokensOnOtherNetworks)
+  const showAdd = canAdd && !hasDepositToken
+  const showAddMore =
+    canAdd && hasDepositToken && !(hasTokensOnSameNetwork && hasTokensOnOtherNetworks)
+  const showTransfer =
+    (!hasDepositToken && !hasTokensOnSameNetwork && !hasTokensOnOtherNetworks) ||
+    (!canAdd && !(hasDepositToken && hasTokensOnSameNetwork && hasTokensOnOtherNetworks))
```

Comment:
Show transfer instead of add if the token is:
- Not a cash-in token
- Not [this case](https://www.figma.com/design/E1rC3MG74qEg5V4tvbeUnU/Earn?node-id=8300-99281&node-type=instance&t=dnj7oAE5JsyFaUFa-0) (has deposit token, same chain and different chain tokens)